apps/prairielearn/src/tests/manualGrading.test.ts

```diff
@@ -1025,10 +1025,10 @@ describe('Manual Grading', function () {
           body: new URLSearchParams({
             __action: 'set_time_limit_all',
             __csrf_token: token,
-            plus_minus: 'unlimited',
+            action: 'unlimited',
             time_add: '0',
-            time_ref: 'minutes',
             reopen_closed: 'on',
+            date: new Date().toISOString().slice(0, 10),
```

Comment:
Is this necessary? Can it just be omitted?

Reply:
Removed in [04498d3](https://github.com/PrairieLearn/PrairieLearn/pull/10800/commits/04498d322330eb9d5e4dfc31e76260a6d9709196).